tests/App/AppKernel.php

```diff
@@ -17,7 +17,7 @@
 use Sonata\AdminBundle\SonataAdminBundle;
 use Sonata\BlockBundle\SonataBlockBundle;
 use Sonata\Doctrine\Bridge\Symfony\Bundle\SonataDoctrineBundle;
-use Sonata\Form\Bridge\Symfony\SonataFormBundle;
+use Sonata\Form\Bridge\Symfony\Bundle\SonataFormBundle;
```

Comment:
This should not be added, maybe we need to bump sonata form bundle in composer.json